### PR TITLE
Use common versions of commons-codec and commons-logging

### PR DESCRIPTION
**Pull request by tevans78**
#build 

This PR creates new common bundles for commons-codec and commons-logging which use the latest versions. Next step is to remove the older versions but WebSphere Liberty changes may be required first.

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -73,13 +73,15 @@ commons-cli:commons-cli:1.4
...
 commons-codec:commons-codec:1.14
+commons-codec:commons-codec:1.15
 commons-codec:commons-codec:1.3
 commons-codec:commons-codec:1.6
 commons-collections:commons-collections:3.2.2
 commons-io:commons-io:2.8.0
 commons-lang:commons-lang:2.4
 commons-logging:commons-logging:1.1.1
 commons-logging:commons-logging:1.1.3
+commons-logging:commons-logging:1.2
```
Comment (Azquelt) [inline dev/cnf/oss_dependencies.maven:84]: Given that the goal of the PR is to use common versions, can the other versions be removed from this file?

Comment (tevans78) [reply dev/cnf/oss_dependencies.maven:84]: going to do removal in a third PR, after I have switched CL to use the new bundles
1. Introduce new versions in OL and use common bundles
2. Switch CL to use common bundles where possible
3. Delete old versions from OL

File: dev/com.ibm.ws.jsp.2.3_fat/build.gradle [hunk 2]
```diff
@@ -9,9 +9,9 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 dependencies {
-    requiredLibs 'commons-codec:commons-codec:1.11',
-      'commons-codec:commons-codec:1.6',
-      'commons-logging:commons-logging:1.1.3',
+    requiredLibs project(':io.openliberty.org.apache.commons.codec'),
+      project(':io.openliberty.org.apache.commons.codec'),
+      project(':io.openliberty.org.apache.commons.logging'),
```
Comment (Azquelt) [inline dev/com.ibm.ws.jsp.2.3_fat/build.gradle:14]: It seems odd that this fat had two versions of commons-codec in the first place, but it certainly doesn't now need the same version declared twice.

File: dev/com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle [hunk 3]
```diff
@@ -29,8 +29,6 @@ dependencies {
    */
   requiredLibs project(':com.ibm.ws.security.saml.sso_fat.common'),
                project(':com.ibm.ws.org.apache.httpcomponents'),
-               'commons-codec:commons-codec:1.4',
-               'commons-logging:commons-logging:1.1.1',
                'httpunit:httpunit:1.6.2',                           // 1.7 has issues noted above
```
Comment (Azquelt) [inline dev/com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle:32]: I guess this project just didn't really require these libraries at all?

Comment (tevans78) [reply dev/com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle:32]: hmmm i'm not sure what happened there ... should have just been switched to the new projects

File: dev/com.ibm.ws.webcontainer.servlet.3.1_fat/build.gradle [hunk 4]
```diff
@@ -20,12 +20,12 @@ dependencies {
...
-      'commons-codec:commons-codec:1.11',
+      project(':io.openliberty.org.apache.commons.codec'),
       'org.apache.httpcomponents:httpmime:4.5.3',
       project(':com.ibm.ws.org.apache.httpcomponents'),
       'org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627',
-      'commons-codec:commons-codec:1.6',
-      'commons-logging:commons-logging:1.1.3',
+      project(':io.openliberty.org.apache.commons.codec'),
+      project(':io.openliberty.org.apache.commons.logging'),
       'xml-apis:xml-apis:1.4.01',
```
Comment (Azquelt) [inline dev/com.ibm.ws.webcontainer.servlet.3.1_fat/build.gradle:29]: Double dependency on commons.codec

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.2/bnd.bnd [hunk 5]
```diff
@@ -30,7 +30,6 @@ tested.features: jsp-2.3, jdbc-4.1, el-3.0, federatedregistry-1.0, ldapregistry-
 
 # net.sourceforge.htmlunit:htmlunit for com.gargoylesoftware.htmlunit
 # not used com.ibm.ws.security_fat.common.tooling;version=latest,\
-#'commons-codec:commons-codec:1.6',
```
Comment (Azquelt) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.2/bnd.bnd:33]: Weird comment change.

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/bnd.bnd [hunk 6]
```diff
@@ -37,7 +37,6 @@ tested.features: jsp-2.3, jdbc-4.1, el-3.0, appsecurity-3.0, servlet-4.0, jaxb-2
 
 # net.sourceforge.htmlunit:htmlunit for com.gargoylesoftware.htmlunit
 # not used com.ibm.ws.security_fat.common.tooling;version=latest,\
-#'commons-codec:commons-codec:1.6',
```
Comment (Azquelt) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/bnd.bnd:40]: another weird comment change

**Review by Azquelt — APPROVED** [review]
I've left comments but I don't think anything I've noted is a serious issue, given that the build has been run and the failures accounted for.

Any changes can be done before merging or as a follow up

**Comment by tevans78** [comment]
#run-libby-bot

**Pull request merged**